README.md

```diff
@@ -153,14 +153,12 @@ Create or join a [discussion](https://github.com/PowerShell/PowerShell/discussio
 
 Want to chat with other members of the PowerShell community?
 
-We have a Gitter Room which you can join below.
-
-[![Join the chat](https://img.shields.io/static/v1.svg?label=chat&message=on%20gitter&color=informational&logo=gitter)](https://gitter.im/PowerShell/PowerShell?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
-
-There is also the community-driven PowerShell Virtual User Group, which you can join on:
+We have a dozens of topic specific channels on our community-driven PowerShell Virtual User Group, which you can join on:
 
+* [Gitter](https://gitter.im/PowerShell/PowerShell)
```

Comment:
Are these all connected or separate independent discussion forums?

Reply:
I know that Discord/IRC/Slack are all linked together, not sure about Gitter though. @Jaykul would need to reply regarding that one.

Reply:
Yep, Discord and Slack are fully bridged. Some of the gitter channels are bridged, some are not. I would be very happy to bridge them all, but years ago, when gitter first appeared, I was asked to only bridge a few at first... and I never went back and pursued the rest. 

I think it would be good to have at least all the official project gitter channels plumbed into the community, especially since Gitter is Matrix now, so we can do that with _style_ instead of the weird way they're plumbed now. 😁

On IRC there has historically only been the one channel (for several reasons), but there will definitely be more now (starting with `#powershell-linux`)